postgres-appliance/bootstrap/clone_with_wale.py

```diff
@@ -65,7 +67,7 @@ def fix_output(output):
             yield '\t'.join(line.split())
 
 
-def choose_backup(backup_list, recovery_target_time):
+def choose_backup(backup_list, recovery_target_time, recovery_target_timeline):
```

Comment:
How do you use this newly added parameter to choose the latest backup file? It seems `recovery_target_timeline` argument is not used by the function

Reply:
Hey, yes - so we wanted to add a new parameter "timeline id" for the Zalando Operator and Spilo. And then we realised there is a bug that Spilo always restores to the backup closest to the provided timestamp irrespective of the timeline. Which means that its possible that a restore is done from an older timeline if the backup closest to the timestamp was on that timeline - which would be a wrong scenario in case restore is required from the latest timeline. So apart from the current PRs I will create a bug on this and open a PR to it.